src/components/Auth/AuthForm.spec.js

```diff
@@ -40,46 +40,48 @@ describe('AuthForm', () => {
 });
 
 describe('Signup', () => {
-  it('Register a new user on the signup form', async () => {
-    const { getByText, getByLabelText } = render(
-      <BrowserRouter>
-        <SignUpForm />
-      </BrowserRouter>
-    );
+  fit('Register a new user on the signup form', async () => {
```

Comment:
fit? is this intended? :D

Reply:
Yeah, that was so I could test a single test at a time :) I pushed up a commit... one sec!

Reply:
I applied https://github.com/codebuddies/frontend/pull/111/commits/bf9c00b987bc3d9a50ab1b11ebdcd5654cc7cbba based on @gauravchl's suggestion in https://github.com/codebuddies/frontend/pull/105/commits/89b51566816ab1255d3376bd7bc0f36065cacc3b, but the interesting thing here is that if I run each test using `fit` individually, the tests pass without those warnings. 

But if I run `npm run test` with that commit in place, I still see: 
![image](https://user-images.githubusercontent.com/4512699/80303188-7dc81c80-8763-11ea-9115-bbbd5e76d9d4.png)

Not sure if there is something I'm obvious I missed, but tldr I'm very surprised to still see that error if I don't run each test individually...

Reply:
I recorded a quick screenrecording about the problem: https://gitduck.com/watch/5ea551ce1f9488f393e05e1a